e2e/cli-e2e/mocks/fs.mock.ts

```diff
@@ -0,0 +1,29 @@
+import { mkdirSync, rmSync, writeFileSync } from 'fs';
+import { join } from 'path';
+
+export function cleanFolder<T extends object>(
+  dirName = 'tmp',
+  content?: { [key in keyof T]: string },
+) {
+  rmSync(dirName, { recursive: true, force: true });
+  mkdirSync(dirName);
+  if (content) {
+    for (const fileName in content) {
+      writeFileSync(join(dirName, fileName), content[fileName]);
+    }
+  }
+}
+
+export function cleanFolderPutGitKeep<T extends object>(
+  dirName = 'tmp',
+  content?: { [key in keyof T]: string },
+) {
+  rmSync(dirName, { recursive: true, force: true });
+  mkdirSync(dirName);
+  writeFileSync(join(dirName, '.gitkeep'), '');
+  if (content) {
+    for (const fileName in content) {
+      writeFileSync(join(dirName, fileName), content[fileName]);
+    }
+  }
+}
```

Comment:
I don't really see the point of these functions. The `tmp` directory is already being automatically cleaned between tests in [`global-setup.ts`](https://github.com/flowup/quality-metrics-cli/blob/main/global-setup.ts). If you want a `.gitkeep` file, you can add it there. I think that's a much better solution that duplicating it everywhere.

- Vitest docs
  - [globalSetup](https://vitest.dev/config/#globalsetup) - that's what each project should be using now
  - [setupFiles](https://vitest.dev/config/#setupfiles) - we can use this as well (advantage is you can add `beforeEach` blocks, etc.)

Reply:
The globalSetup is executed only once. I have overlapping files in between tests. 

I opened up a new issue to use vitest setupFile as I'm not familiar ATM.
https://github.com/flowup/quality-metrics-portal/issues/119